app/services/finance/ecf/assurance_report/presenter.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Finance
+  module ECF
+    class AssuranceReport
+      class Presenter
```

Comment:
Suggestion: I would prefer if we had a CSV serialiser, for instance `AssuranceRerpotCsvSerializer` which accepts the scope from the query and generates the CSV headers and rows (without the need to include the lib version). This means all the CSV logic lives in that serialiser, and we follow the established pattern we have for other existing CSV serialisers in the API, what do you think?